src/transformers/models/beit/image_processing_beit.py

```diff
@@ -14,12 +14,11 @@
 # limitations under the License.
 """Image processor class for Beit."""
 
-import warnings
 from typing import Any, Dict, List, Optional, Tuple, Union
 
 import numpy as np
 
-from ...image_processing_utils import BaseImageProcessor, BatchFeature, get_size_dict
+from ...image_processing_utils import INIT_SERVICE_KWARGS, BaseImageProcessor, BatchFeature, get_size_dict
```

Comment:
Restored from config image processors propogate `image_processor_type` and `processor_type` named arguments to `__init__` method. Despite they are not necessary and image_processor can be initialized without these arguments, some test logic relies on them. I tried to remove them, but it looks like too many models should be touched, probably, it's better to try as a follow-up. 
For now, I came up with this suboptimal solution, just to specify them as extra names for `@filter_out_non_signature_kwargs(extra=INIT_SERVICE_KWARGS)`

Reply:
I think this a pretty clean and pragmatic solution